New package: HJSON v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: HJSON
- Repository: https://github.com/fmatesa/HJSON.jl
- Created by: @fmatesa
- Version: v1.0.0
- Commit: da3f6a6dd03332a8bbab8c9957ee515108c43b08
- Git reference: HEAD

fmatesa commented:
Hi, I'm getting problems due to the name similarities, but I feel like this name is appropriate for what the package does.

JuliaHubRegistrator committed:
New package: HJSON v1.0.0

UUID: 6226b997-59fe-41cb-ba3d-177c22dd73bb
Repo: https://github.com/fmatesa/hjson.jl.git
Tree: 85dcc33b0b94883260a49e4844afdb8fe16e340d

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

fmatesa commented:
> The name is fine, but
> 
> > Repo URL does not end with /name.jl.git, where name is the package name
> 
> needs to be addressed. And then, it will need a manual merge because of the all-caps name

Repo URL should be fixed now.

goerz commented:
Unfortunately, I think you have to trigger a new registration after renaming the repo
